docs/best_practices/repositories.rst

```diff
@@ -24,8 +24,26 @@ repositories in a structure similar to the following:
 The highest level directory contains only a few folders for the major types of
 Galaxy repositories; tools, packages, data managers, and sometimes visualizations and datatypes.
 
-Packages
---------
+ToolShed Repositories
+---------------------
+
+A Github repository many correspond to any number of published ToolShed
+repositories.
+
+Every unrestricted tool shed repository should contain a README_ file -
```

Comment:
Once https://github.com/galaxyproject/planemo/pull/134 gets merged these repositories on the source/GitHub side will be able to have READMEs and they will just be filtered out on ``shed_upload`` - it may be worth just not making a distinction here (I am for saying everything should have a README in github).

As for defining every term - is this really meant to replace ToolShed documentation? I am fine with that but it would be huge undertaking.